automation_testing/sv-mosaic.config.ts

```diff
@@ -0,0 +1,32 @@
+import { PlaywrightTestConfig } from "@playwright/test";
+
+const config: PlaywrightTestConfig = {
+
+    testDir: 'tests',
+    use: {
+        headless: true,
+        viewport: { width: 1280, height: 720 },
+        actionTimeout: 1500000,
+        ignoreHTTPSErrors: true,
+        //video: 'retain-on-failure',
+        //screenshot: 'only-on-failure'
+        video: 'off',
+        screenshot: 'off'
+    },
+    projects: [
+        {
+            name: 'Chromium',
+            use: { browserName: 'chromium' },
+        },
+        {
+            name: 'Firefox',
+            use: { browserName: 'firefox' },
+        },
+        {
+            name: 'Webkit',
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 3 tabs but found 12 spaces.](https://app.codacy.com/gh/simpleviewinc/sv-mosaic/pullRequest?prid=9498610)
```suggestion
			name: 'Webkit',
```